app/src/components/LegendOther.tsx

```diff
@@ -0,0 +1,31 @@
+import "./Legend.css"
+import { Spacer } from "./Spacer"
+
+interface LegendOtherProps {
+  items: string[]
+  show: boolean
+}
+
+export function LegendOther(props: LegendOtherProps) {
+  if (!props.show) return null
+
+  return (
+    <div className="legend-entry">
+      {props.items.map((legendItem, i) => {
+        let [, color] = legendItem.split("|")
```

Comment:
Could this be refactored to use an array, so splitting is not necessary

Reply:
Could be, but it might require us to implement a new class.